cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -585,14 +690,16 @@ public:
               ::cuda::std::is_void_v<PolicyHub>,
               /* fallback_policy_hub */
               detail::histogram::policy_hub<SampleT, CounterT, NUM_CHANNELS, NUM_ACTIVE_CHANNELS, /* isEven */ 0>,
-              PolicyHub>::MaxPolicy>
-  CUB_RUNTIME_FUNCTION static cudaError_t DispatchRange(
+              PolicyHub>::MaxPolicy,
+            typename NumOutputLevelsArrayT = ::cuda::std::array<int, NUM_ACTIVE_CHANNELS>,
+            typename LevelsArrayT          = ::cuda::std::array<const LevelT*, NUM_ACTIVE_CHANNELS>>
+  CUB_RUNTIME_FUNCTION static cudaError_t DispatchRangeDeviceInit(
```

Comment:
Suggestion: please add new APIs below the existing ones so I have an easier time seeing whether there are any diffs on the existing APIS.